crates/astria-sequencer/src/app/mod.rs

```diff
@@ -961,7 +961,7 @@ impl App {
     /// Executes a signed transaction.
     #[instrument(name = "App::execute_transaction", skip_all, fields(
         signed_transaction_hash = %telemetry::display::base64(&signed_tx.sha256_of_proto_encoding()),
-        sender = %signed_tx.verification_key().address(),
+        sender = %telemetry::display::base64(&signed_tx.verification_key().address_bytes()),
```

Comment:
should this display as bech32?

Reply:
Can't right now because all we have is the sender (and hence the verification key), but no prefix, and hence no bech32m.

I am not sure how to address this at this point.